draft-ietf-tls-tls13.md

```diff
@@ -3546,35 +3546,32 @@ that remains is to calculate the key schedule.
 
 The TLS handshake establishes one or more input secrets which
 are combined to create the actual working keying material, as detailed
-below. The key derivation process makes use of the following functions,
-based on HKDF {{RFC5869}}:
+below.
+The key derivation process makes use of the HKDF-Extract and HKDF-Expand
+functions as defined for HKDF {{RFC5869}}, as well as the functions
+defined below:
 
 ~~~~
-  HKDF-Extract(Salt, IKM) as defined in {{RFC5869}}.
-  
-  HKDF-Expand-Label(Secret, Label, Messages, Length) =
+  HKDF-Expand-Label(Secret, Label, HashValue, Length) =
```

Comment:
This is a regression, I think.  I think that this should take Messages (a list of messages) as input still, but then define that HkdfLabel.hash_value is H(concat(Messages))